packages/dds/tree/src/feature-libraries/sequence-field/compose.ts

```diff
@@ -438,7 +533,6 @@ export class ComposeQueue<T> {
         newMarks: Changeset<T>,
         genId: IdAllocator,
         moveEffects: MoveEffectTable<T>,
-        reassignNewMoveIds: boolean = true,
         updatePairedMarkStatus: boolean = true,
```

Comment:
May as well remove this parameter. Otherwise someone my pass in `false` and wonder what's happening.